GitUI/UserControls/FileStatusList.cs

```diff
@@ -488,7 +488,7 @@ public int UnfilteredItemsCount()
             }
             else
             {
-                return GitItemStatusesWithParents.SelectMany(pair => pair.Value).Count();
+                return GitItemStatusesWithParents.Sum(pair => pair.Value.Count);
```

Comment:
Are you sure the change is equivalent to the original?
`SelectMany` returns an `IEnumerable<whatever>` then does a count over the collection. The proposal sums up counts...
I'm looking at in a browser, and it is not exactly obvious.

Reply:
I believe it's equivalent but faster. Let's look at an example.

```c#
var source = new int[][]
{
    Enumerable.Range(1000).ToArray(),
    Enumerable.Range(5000).ToArray()
};

// flattens to a sequence and enumerates 6000 items

source.SelectMany(a => a).Count() == 6000

// enumerates two items, sums 1000 and 5000

source.Sum(a => a.Length) == 6000
```

So yes, it's the same but less computationally expensive. Especially as `SelectMany` returns a collection for which the usual `Enumerable.Count` performance short-cuts (testing for `ICollection.Count`) do not apply due to its laziness.

The magnitude of gain in this change would be less than in my contrived example, as many items will have one parent. But there is still less iteration going on overall due to the reduction in number of operators.